commands/registerCommands.go

```diff
@@ -49,6 +49,7 @@ func Register(s *discordgo.Session) {
 	command("help", "displays this message", help, false)
 	command("register", "registers you as a member of the server", serverRegister, false)
 	command("online", "see how many people are online in minecraft.netsoc.co", online, false)
+	command("dig", "run a DNS query: dig TYPE DOMAIN [@RESOLVER]", digCommand, false)
```

Comment:
do we want to make this a committee command?

Reply:
let people use it in the public server for the meme